py3status/core.py

```diff
@@ -326,7 +326,7 @@ def setup(self):
             # load and spawn i3status.conf configured modules threads
             self.load_modules(self.py3_modules, user_modules)
 
-    def notify_user(self, msg, level='error', rate_limit=None, key=''):
+    def notify_user(self, msg, level='error', rate_limit=None, module_name=''):
```

Comment:
It wouldn't actually end up clashing because `None` doesn't compare equal to the string `'None'`. I'd be in favor of changing just because `None` is more Pythonistic.